tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/annotations/DeprecationAnnotationTest.java

```diff
@@ -36,7 +36,7 @@ public class DeprecationAnnotationTest {
     @Test(description = "Test the deprecation annotation")
     public void testDeprecationAnnotation() {
         CompileResult compileResult = BCompileUtil.compile("test-src/annotations/deprecation_annotation.bal");
-        Assert.assertEquals(compileResult.getWarnCount(), 48);
+        Assert.assertEquals(compileResult.getWarnCount(), 53);
```

Comment:
Let's move this assertion to the end and assert against `i`.

Reply:
Updated 3fc30db5fbc84e389a635fa4c84d2b913b7b6035